backend/api/backend/gallery/gallery.file.ts

```diff
@@ -40,32 +40,26 @@ export class GalleryFile {
     return isDir.isDirectory();
   }
 
-  // Returns number of pages for all images.
-  async getTotalPages(): Promise<number> {
-    const filesAmount = await this.getFilesAmountFromDb();
-    const onePage = 1;
-    if (filesAmount <= PER_PAGE) return onePage;
-
-    const remainder = filesAmount % PER_PAGE;
-    if (remainder === 0) return filesAmount / PER_PAGE;
-
-    return Math.trunc(filesAmount / PER_PAGE) + onePage;
+  async getNumberOfPages(limit: number): Promise<number> {
+    const total = await this.getFilesAmountFromDb();
```

Comment:
I think it's not the best name for variable if you don't mean to store information about machine oil here) moreover, you don't use 'total' anywhere but inside calculatePagesNumber. so you can move calculation of 'total' inside calculatePagesNumber.